.goreleaser.yml

```diff
@@ -40,7 +44,8 @@ builds:
   ldflags:
   # one-line ldflags to bypass the goreleaser bugs
   # the git tree state is guaranteed to be clean by goreleaser
-  - -w -X oras.land/oras/internal/version.Version={{.Version}} -X oras.land/oras/internal/version.GitCommit={{.FullCommit}} -X oras.land/oras/internal/version.BuildMetadata= -X oras.land/oras/internal/version.GitTreeState=clean
+  - -w -s -buildid= -X oras.land/oras/internal/version.Version={{.Version}} -X oras.land/oras/internal/version.GitCommit={{.FullCommit}} -X oras.land/oras/internal/version.BuildMetadata= -X oras.land/oras/internal/version.GitTreeState=clean
```

Comment:
nit: just curious on the reason of omitting the golang toolchain build ID.

ref: https://pkg.go.dev/cmd/link

Reply:
Is it for reproducible build as I've noticed the `mod_timestamp`?

Reply:
Yes, this is for reproducible builds.